### ISSUE TITLE
Device: Frequent connection failures

### ISSUE THREAD

**Issue by danlesbm** [frangoteam/FUXA#126]
Frequent connection failures if set timeout is short. Specifically: if left less than 30 seconds, there will be errors
![image](https://user-images.githubusercontent.com/65875578/113511682-3bba7400-958b-11eb-9f3e-3234cb8663be.png)

**Comment by unocelli** [comment]
I assume this screenshot is not relevant...with which type of connection? can you send me the exception message?

**Comment by danlesbm** [comment]
> I assume this screenshot is not relevant...with which type of connection? can you send me the exception message?

OK. I uploaded the wrong picture. below is the video, if left less than 30 seconds or after reboot, there will be errors

https://user-images.githubusercontent.com/65875578/113951901-be1e8e80-983e-11eb-9911-6a7add254214.mp4

**Comment by unocelli** [comment]
Thanks, i found a concept error, after a request error don't try again, I fixed and i added the missing error log.
can you try and send me log file '_log/fuxa-err.log'

**Comment by danlesbm** [comment]
> Thanks, i found a concept error, after a request error don't try again, I fixed and i added the missing error log.
> can you try and send me log file '_log/fuxa-err.log'

ok

**Comment by danlesbm** [comment]
It still didn't run. It running, i reboot my PC, after reconnect but didn't run. I must push modify then push ok --> it run.
I didn't find file log

**Comment by unocelli** [comment]
I'm not sure if you did the update. 
It's better if you test the WebAPI alone,  install FUXA again from [master branch](https://github.com/frangoteam/FUXA/archive/refs/heads/master.zip) or 'git clone https://github.com/frangoteam/FUXA'
start the server, configure only the WebAPI and then you have to find the error log in the folder where you have started the server 'FUXA/server/_logs'.

**Comment by danlesbm** [comment]
> I'm not sure if you did the update.
> It's better if you test the WebAPI alone, install FUXA again from [master branch](https://github.com/frangoteam/FUXA/archive/refs/heads/master.zip) or 'git clone https://github.com/frangoteam/FUXA'
> start the server, configure only the WebAPI and then you have to find the error log in the folder where you have started the server 'FUXA/server/_logs'.

yes of course, i ran it with command: "npm install -g --unsafe-perm @frangoteam/fuxa"

**Comment by unocelli** [comment]
By npm server you install only the released version and I have not yet released with the latest changes.
To be update you have to install from github

**Comment by danlesbm** [comment]
> By npm server you install only the released version and I have not yet released with the latest changes.
> To be update you have to install from github

OK. Let me try again

**Comment by unocelli** [comment]
You have to: 
- download from [master branch](https://github.com/frangoteam/FUXA/archive/refs/heads/master.zip)
- cd ./server
- npm install
- npm start

**Comment by danlesbm** [comment]
> You have to:
> 
> * download from [master branch](https://github.com/frangoteam/FUXA/archive/refs/heads/master.zip)
> * cd ./server
> * npm install
> * npm start

I tested, it's ok

**Comment by unocelli** [comment]
great, you can close the solved issues
